lms/templates/admin/macros.jinja

```diff
@@ -0,0 +1,131 @@
+{% macro field_body(label) %}
```

Comment:
I don't think you have to put all the macros in here. You could keep the common ones and put things which are only used in one file like `registrations_fields` in that file.

Otherwise we introduce distance between the definition and the usage without any other benefit (like re-use).